extension.js

```diff
@@ -17,7 +17,7 @@ function goToLastWorkspace() {
     return;
   }
 
-  let ws = global.screen.get_workspace_by_index(lastWorkspace);
```

Comment:
Can you try to keep it BC?

It wouldn't be too hard to keep backward compatibility by using the construct `(global.workspace_manager || global.screen).get…`, right?

Reply:
Added your suggestions. It works in 3.28 as well now.